media-libs/mesa/mesa-9999.ebuild

```diff
@@ -123,7 +123,7 @@ RDEPEND="${RDEPEND}
 # How to use it:
 # 1. Specify LLVM_MAX_SLOT (inclusive), e.g. 16.
 # 2. Specify LLVM_MIN_SLOT (inclusive), e.g. 15.
-LLVM_MAX_SLOT="16"
+LLVM_MAX_SLOT="17"
```

Comment:
It's not ready, I'm afraid. It wasn't added for a reason.

https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/2153 and https://bugs.gentoo.org/914849.

Reply:
If 23.3 is fine, can you give any sources for what changed?

Reply:
Ah sorry I'm AMD only here and I've been using 17 since it's release

Reply:
Can we maybe get something like mesa_llvm_slot_17 use expand flags in the longterm, to allow easy override instead of current semi-automagic llvm slot selection?